lib/versioning/pvp/index.ts

```diff
@@ -0,0 +1,80 @@
+import { regEx } from '../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../loose/generic';
+import { VersioningApi } from '../types';
+
+export const id = 'pvp';
+export const displayName = 'PVP';
+export const urls = ['https://pvp.haskell.org'];
+export const supportsRanges = false;
+
+/**
+ * At least 3 components and no leading 0s
+ * https://pvp.haskell.org/#version-number
+ */
+const versionPattern = regEx(
+  /^((([0-9])|([1-9][0-9]*))\.){2,}(([0-9])|([1-9][0-9]*))$/
+);
+
+class PvpVersioningApi extends GenericVersioningApi {
+  /**
+   * PVP has two major components A.B
+   * To keep compatability with Renovate's versioning API we will treat it as a float
```

Comment:
```suggestion
   * To keep compatibility with Renovate's versioning API we will treat it as a float
```

Fix a typo in the comment. 😉